### PR TITLE
Task #12 Add Halstead Volume metric calculation

### PR DESCRIPTION
**Pull request by zuoqin**
Add Hasltead volume metric automatic calculation

### CODE REVIEW

**Comment by 0crat** [comment]
@zuoqin[/z](https://www.0crat.com/u/zuoqin) this pull request is too small, just 5 lines changed (less than 10), there will be no formal code review, see [§53](http://www.zerocracy.com/policy.html#53) and [§28](http://www.zerocracy.com/policy.html#28); in the future, try to make sure your pull requests are not too small; @yegor256[/z](https://www.0crat.com/u/yegor256) please review this and merge or reject

File: scripts/02-calculate-metrics.py [hunk 1]
```diff
@@ -45,16 +45,19 @@ def process_file(file):
...
         loc = m.value()
         m = NPathMetric(file[:-1])
         npath = m.value(False)['data'][0]['complexity']
+        m = HVMetric(file[:-1])
+        hv = m.value()['data'][0]['halsteadvolume']
         lock_path = '{}.lock'.format(OUT_FILE_NAME)
         lock = FileLock(lock_path, timeout=10)
         with lock:
-            open(OUT_FILE_NAME, 'a').write('{};{};{};{}\n'.format(file[:-1], cc, loc, npath))
+            open(OUT_FILE_NAME, 'a').write('{};{};{};{};{}\n'.format(file[:-1], cc, loc, npath, hv))
```
Comment (vryazanov) [inline scripts/02-calculate-metrics.py]:
```
    with FileLock(lock_path, timeout=10), open(OUT_FILE_NAME, 'a') as fin:
        fin.write('{};{};{};{};{}\n'.format(file[:-1], cc, loc, npath, hv))
```
it's more pythonic, I believe :)

**Comment by yegor256** [comment]
@zuoqin something is wrong with `test/metrics/entropy/sample-1.java`. Better add it to `.pdd` config.

**Comment by yegor256** [comment]
@zuoqin not required anymore

**Comment by 0crat** [comment]
Job `gh:yegor256/aibolit#25` is not assigned, can't get performer

**Comment by 0crat** [comment]
This job is not in scope

**Pull request closed without merging**